saleor/graphql/core/scalars.py

```diff
@@ -88,3 +88,23 @@ def parse_literal_object(node):
             if field.name.value == "unit":
                 unit = field.value.value
         return Weight(**{unit: value})
+
+
+class UUID(graphene.UUID):
+    @staticmethod
+    def serialize(uuid):
+        return super(UUID, UUID).serialize(uuid)
```

Comment:
`super().serialize(uuid)` won't work?

Reply:
Yes, in Python 3 we should use `super` without arguments as @korycins suggests. It should be changed here and in methods below as well.

Reply:
@korycins @maarcingebala 
Well this was new for me too. According to this thread [https://bugs.python.org/issue31118](https://bugs.python.org/issue31118) super without arguments does not work with static methods by design. There are two possible solutions.
1. Pass class as I did above for both super arguments
2. Use `graphene.UUID.method(...)`
For me both are fine.

Reply:
If the "Python 3 approach" doesn't work here I'm fine with the current solution.